src/librustc_data_structures/dfx.rs

```diff
@@ -0,0 +1,54 @@
+pub use rustc_hash::FxHashMap;
+use std::borrow::Borrow;
+use std::hash::Hash;
+
+
+/// A wrapper around FxHashMap that does not provide iteration support.
```

Comment:
It'd be good to mention "deterministic" here somewhere.

Reply:
Thanks, mark. Shall add it.